oat-sa/extension-tao-mediamanager
# Fix/bundle paths

Fixed path to bundles.

How to test

- Switch to branch
- Switch off debug mode in `config/generis.conf.php`
- `define('DEBUG_MODE', false);`
- Open Assets, go to Authoring, then Preview
- Should be no errors
